release/models/acl/openconfig-packet-match.yang

```diff
@@ -394,12 +394,32 @@ module openconfig-packet-match {
         "Destination port or range";
     }
 
-    leaf-list tcp-flags {
-      type identityref {
-        base oc-pkt-match-types:TCP_FLAGS;
+    container transport-details {
+      choice explicit-or-builtin {
+        case explicit-flags {
+          leaf-list tcp-flags {
+            type identityref {
+              base oc-pkt-match-types:TCP_FLAGS;
+            }
+            description
+              "List of TCP flags to match";
+          }
+          leaf match {
+            type enumeration {
+              enum ANY;
+              enum ALL;
```

Comment:
Please add a description for each of these options, and the `match` leaf itself.